NuRadioMC/SignalGen/emitter.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+import numpy as np
+from NuRadioReco.utilities import units, fft
+from NuRadioMC.SignalGen import parametrizations as par
+import logging
+logger = logging.getLogger("SignalGen.askaryan")
+
+
+def set_log_level(level):
+    logger.setLevel(level)
+    par.set_log_level(level)
```

Comment:
Is this line needed?

Reply:
good point. These lines should be removed.